e2e/addons/addons_test.go

```diff
@@ -57,9 +57,36 @@ var _ = Describe("addons flow", func() {
 			})
 		})
 
-		It("env init should succeed", func() {
+		It("should succeed", func() {
 			Expect(testEnvInitErr).NotTo(HaveOccurred())
 		})
+
+		It("should create environment manifest", func() {
+			Expect("./copilot/environments/test/manifest.yml").Should(BeAnExistingFile())
+		})
+
+		It("should show only bootstrap resources in env show", func() {
+			envShowOutput, envShowError := cli.EnvShow(&client.EnvShowRequest{
+				AppName: appName,
+				EnvName: "test",
+			})
+			Expect(envShowError).NotTo(HaveOccurred())
+			Expect(len(envShowOutput.Resources)).To(Equal(2)) // Contains only bootstrap resources - two IAM roles.
+		})
+	})
+
+	Context("when deploying the environment", func() {
+		var envDeployErr error
+		BeforeAll(func() {
+			_, envDeployErr = cli.EnvDeploy(&client.EnvDeployRequest{
+				AppName: appName,
+				Name:    "test",
+			})
+		})
+
+		It("should succeed", func() {
+			Expect(envDeployErr).NotTo(HaveOccurred())
+		})
 	})
 
```

Comment:
Should we run `env show` again to see more resources?

Reply:
(In just one of the e2e tests, not necessarily this one.)

Reply:
added!